packages/cadl-rlc-emitter/src/transform/transformResponses.ts

```diff
@@ -33,19 +37,20 @@ export function transformToResponseTypes(program: Program, importDetails: Map<Im
     return rlcResponses;
 }
 
-function transformHeaders(from: HttpOperationResponse, to: ResponseMetadata) {
-    if (!from.responses.length) {
+function transformHeaders(
+    response: HttpOperationResponse): ResponseHeaderSchema[] | undefined {
+    if (!response.responses.length) {
         return;
-    } else if (from.responses.length > 1) {
+    } else if (response.responses.length > 1) {
         // TODO: handle one status code map to multiple rsps
     }
-    const headers = from.responses[0]?.headers;
+    const headers = response.responses[0]?.headers;
```

Comment:
Why we don't need other responses' headers , just the first one ?

Reply:
@joheredi How do you think below case? Do you think we need to adjust our response structure?
This is case we need to do further investigation between CADL and Swagger. 

#### Swagger Response Structure
One operation_id has different responses
However in the same response we will share the status code, body and headers etc
#### CADL Response Structure
One operation_id has different responses
In the same response we'll share the status code but it may have diff body and headers(e.g content-type)
#### RLC Current Response
Currently RLC response has similar structure with swagger one so we can't handle the different bodies with below case.
```
"responses": {
          "200": {
            "description": "The request has succeeded.",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/Widget"
                },
              "text/plain": {
                "schema": {
                  "$ref": "#/components/schemas/Widget2"
                }
              }
            }
          },
```

Reply:
It looks like we should make our RLC model able to support multiple response types depending on the statusCode, which seems to be an improvement from CADL

Reply:
@joheredi  @qiaozha  I create an issue: https://github.com/Azure/autorest.typescript/issues/1486. I prefer we do more investigation after we have basic version of other files. 

How do you think?